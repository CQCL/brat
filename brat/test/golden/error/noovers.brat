f(Int) -> []
f(a, b) = []
