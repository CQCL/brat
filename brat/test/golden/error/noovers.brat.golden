Error in test/golden/error/noovers.brat@FC {start = Pos {line = 1, col = 1}, end = Pos {line = 2, col = 13}}:
f(Int) -> []
^^^^^^^^^^^^
f(a, b) = []
^^^^^^^^^^^^

  Unification error: Pattern doesn't match expected length for constructor args

